appveyor.yml

```diff
@@ -79,10 +79,6 @@ install:
     - cmd: conda install -n root --quiet --yes conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
 
-    - cmd: # install conda-build 2.x to build with a long prefix
-    - cmd: conda install --yes --quiet conda-build=2
-    - cmd: conda info
```

Comment:
> We should really move everything to conda-build 2 ASAP

I had some issues with conda-build 2 - there were some hardcoded string sizes in `qt` that weren't long enough for the long prefixes. I'll have a better look and post a patch when I get a chance.